packages/main/src/themes/BusyIndicator.css

```diff
@@ -49,17 +49,12 @@
 	align-items: center;
 	position: relative;
 	background-color: inherit;
-	min-width: 8rem;
```

Comment:
here you should add the styles for "Medium", otherwise if the user does not set size attribute, the dots won't be displayed. The framework does not add the default values initially.